go/src/meguca/db/upkeep.go

```diff
@@ -297,3 +298,16 @@ func DeleteBoard(board, by string) error {
 			fmt.Sprintf("board %s deleted by user", board))
 	})
 }
+
+// Allow sluts to #pyu again
+func freePyuLimit() error {
+	_, err := sq.Update("pyu_limit").Set("restricted", false).Exec()
```

Comment:
This will mean it can take up to 1h 59 min for the counter to reset from maxing it out. Maybe run this every 10 minutes or so?

Reply:
Wait what? Shouldn't the hour tasks only take an hour?